### ISSUE TITLE
blurred gui on macOS Catalina 

### ISSUE THREAD

**Issue by murakamiriyugi** [wxMaxima-developers/wxmaxima#1414]
As the title says, everything is blurred on macOS Catalina (please see the attached screenshot). 
System info: wxmaxima 20.09.0, wxwidgets 3.0.5, maxima 5.44.0
WxMaxima was installed using homebrew.

The last non-blurred version was wxmaxima 20.01.2 installed by directly building the app file from the source.

<img width="576" alt="Screen Shot 2020-10-16 at 01 42 07" src="https://user-images.githubusercontent.com/4223730/96167073-6ff49680-0f51-11eb-8836-9d7cbc6a04bb.png">

**Comment by gunterkoenigsmann** [comment]
Are the toolbars blurry, too?

**Comment by murakamiriyugi** [comment]
The toolbars are also blurry. WxMaxima windows (e.g., preferences window) are also blurry. If it might help, the MacBook Pro is 2016, Retina display.

<img width="714" alt="Screen Shot 2020-10-16 at 13 53 38" src="https://user-images.githubusercontent.com/4223730/96218500-7289d700-0fb7-11eb-8e5f-7c4b8cdd222c.png">
<img width="600" alt="Screen Shot 2020-10-16 at 13 55 32" src="https://user-images.githubusercontent.com/4223730/96218505-7584c780-0fb7-11eb-9945-5d1481181bcf.png">

**Comment by gunterkoenigsmann** [comment]
Hmmm... ...then I guess the error is somehow inside the Info.plist file we bundle with the MacOs binary. @MSoegtropIMC : It might have been me who broke this. But - do you have any idea how I managed to do that?

**Comment by MSoegtropIMC** [comment]
Maybe this fixes the menu issue as well - I will do a test build.

**Comment by murakamiriyugi** [comment]
I was able to build the wxmaxima app on my Mac using the modification suggested above but the resulting app crashes when launching the program. As a workaround, I tried changing the segment in src/StreamUtils.cpp file from

: m_locale("en_US.UTF8"),

to

: m_locale("en_US.UTF-8"),

as suggested here: https://github.com/wxMaxima-developers/wxmaxima/issues/1402

The source compiled fine and the main GUI is no longer blurred. But the app uses around 300% of the CPU which heats up my computer significantly. It is effectively unusable now.
